internal/controller/reconciler/reconcile_v2.go

```
@@ -0,0 +1,1331 @@
+			},
+		})
+	}
+
+	return volumes, volumeMounts, nil
+}
+
+func runMigrations(ctx context.Context, client ctrlClient.Client, wandb *apiv2.WeightsAndBiases, manifest serverManifest.Manifest) (ctrl.Result, error) {
+	version := wandb.Spec.Wandb.Version
+
+	if wandb.Status.Wandb.Migration.Ready && wandb.Status.Wandb.Migration.Version == version {
+		for name, _ := range manifest.Migrations {
```

golangci-lint (2.12.2)

[error] S1005: unnecessary assignment to the blank identifier

(staticcheck)